nautobot/ui/src/components/ObjectListTable.js

```diff
@@ -0,0 +1,138 @@
+import { RouterButton } from "./RouterButton";
+import { ButtonGroup } from "@chakra-ui/react";
+import * as Icon from "react-icons/tb";
+import { useLocation } from "react-router-dom";
+import {
+    Box,
+    Heading,
+    NtcThumbnailIcon,
+    MeatballsIcon,
+    Button as UIButton,
+    TableRenderer,
+    useTableRenderer,
+    createColumnHelper,
+    PlusIcon,
+} from "@nautobot/nautobot-ui";
+import { useCallback, useMemo } from "react";
+
+import TableItem from "@components/TableItem";
+
+const getTableItemLink = (idx, obj) => {
+    if (idx === 0) {
+        // TODO: ui-schema should be providing the name of the field used to describe the object (name)
+        return window.location.pathname + obj.id;
+    }
+    if (typeof obj !== "object" || !obj || !obj.url) {
+        return null;
+    }
+    // Remove domain + /api prefix
+    const url = obj.url.replace(window.location.origin + "/api", "");
+
+    // Statuses and Roles should not be linkable
+    if (
+        ["/extras/statuses", "/extras/roles"].some((prefix) =>
+            url.startsWith(prefix)
```

Comment:
url always starts with `/api` if I'm reading the above right, so this seems like dead code anyways?